components/pingfederate.key.manager/src/main/java/org/wso2/pingfederate/model/ClientInfo.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.pingfederate.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * DCR Client Info.
+ */
+public class ClientInfo {
+
+    @SerializedName("name")
+    private String name;
+    @SerializedName("clientId")
+    private String clientId;
+    @SerializedName("redirectUris")
+    private Object redirectUris = new ArrayList();
+    @SerializedName("grantTypes")
+    private Object grantTypes = new ArrayList<>();
+    @SerializedName("description")
+    private String description;
+    @SerializedName("clientAuthnType")
+    private String clientAuthnType;
```

Comment:
is there a meaning for the letter 'n' in  "Authn" ? Is this coming from ping side or a syntax issue in our side?

Reply:
yes this is from ping end